metriq_gym/local/provider.py

```diff
@@ -12,7 +12,29 @@ def __init__(self):
         self._devices: dict[str, LocalAerDevice] = {}
 
     def get_devices(self, **_):
-        return [self.device, *self._devices.values()]
+        devices = [self.device]
+
+        # Try to get IBM backends if QiskitRuntimeService is configured
+        try:
+            service = QiskitRuntimeService()
+            backends = service.backends()
+            # Add available backends to the list
+            for backend in backends:
+                device_id = backend.name
+                if device_id != "aer_simulator":
```

Comment:
Why would service.backends() return aer_simualor anyway? This check looks unnecessary and too defensive.